test/e2e/scalability/density.go

```diff
@@ -328,9 +335,14 @@ func cleanupDensityTest(dtc DensityTestConfig, testPhaseDurations *timer.TestPha
 	}
 	podCleanupPhase.End()
 
+	go framework.GatherApiserverCPUProfile(&profileWG, "DENSITY_deleting_saturation_pods")
+	go framework.GatherApiserverMemoryProfile(&profileWG, "DENSITY_deleting_saturation_pods")
```

Comment:
I would like to grab memory profile at the very end of the test (this would give us total memory allocations).

Reply:
Done.